zio-ftp/src/main/scala/zio/ftp/FtpResource.scala

```diff
@@ -38,7 +39,11 @@ final case class FtpResource(
   lastModified: Long,
   permissions: Set[PosixFilePermission],
   isDirectory: Option[Boolean]
-)
+) {
+
+  def lastModifiedInstant: Instant =
+    Instant.ofEpochSecond(lastModified)
```

Comment:
lastModified is in milliseconds 
```scala
Instant.ofEpochMilli()
```

Reply:
Not on the SFtp server that I was working with today! I was using `ofEpochMilli` and got garbage, then I changed it to `ofEpochSecond` and got the correct results.

Maybe different implementations handle it differently?

Reply:
I've connected to that server using `sftp -v` and it said:
```
debug1: Remote protocol version 2.0, remote software version OpenSSH_7.4
```
So apparently OpenSSH uses second timestamps.

Reply:
I've now tried this with an FTP server, and there it seems that the timestamps are indeed in milliseconds. So we should probably multiply by 1000 in the SFtp implementation and use `ofEpochMilli` here.

Reply:
And since we're switching to strong types anyway, is there any reason to not use a `java.nio.file.Path` for the path?